Add information on modular Kyma

IwonaLanger opened the pull request:
**Description**

Changes proposed in this pull request:

- Add general information on modular Kyma
- Move "Kyma areas" to the main README and Overview README files


**Related issue(s)**
[#17210](https://github.com/kyma-project/kyma/issues/17210)

varbanv commented on docs/01-overview/README.md:
```diff
@@ -8,6 +8,12 @@ Kyma is an application runtime that provides you a flexible and easy way to conn
 
+## Modular Kyma
+
+Classic Kyma offers a fixed set of mandatory components whose development rhythm is synchronized and determined by the release schedule. With the modular approach, Kyma components become modules, each providing one functionality developed independently of the other ones. With each successive release, fewer components will be available within the preconfigured Kyma, but more and more will be offer ...
```
I thought it was possible to only install certain components when you use the CLI?

varbanv commented on docs/README.md:
```diff
@@ -2,6 +2,10 @@ Kyma is an application runtime that provides you a flexible and easy way to conn
 
 ![overview](./assets/kyma-overview.svg)
 
+Classic Kyma offers a fixed set of mandatory components whose development rhythm is synchronized and determined by the release schedule. With the modular approach, Kyma components become modules, each providing one functionality developed independently of the other ones. With each successive release, fewer components will be available within the preconfigured Kyma, but more and more will be offer ...
```
Same comment as the overview readme.

varbanv commented on docs/01-overview/README.md:
```diff
@@ -8,6 +8,12 @@ Kyma is an application runtime that provides you a flexible and easy way to conn
+
+Classic Kyma offers a fixed set of mandatory components whose development rhythm is synchronized and determined by the release schedule. With the modular approach, Kyma components become modules, each providing one functionality developed independently of the other ones. With each successive release, fewer components will be available within the preconfigured Kyma, but more and more will be offer ...
+
+The modular approach boosts Kyma's extensibility and scalability and makes it cost-effective. With fully modularized Kyma, you can choose from many of its modules to facilitate your business needs best.  You can enable or disable modules on demand and configure them, thus determining the costs generated by the infrastructure that your modules use.
```
Kyma is cost-effective, we are improving on that with modularization.  
Maybe even more importantly, we are reducing the attack surface area by not running components that are not needed. We also reduce the upgrade time and risk by not having to upgrade unnecessary components.

IwonaLanger commented on docs/01-overview/README.md:
```diff
@@ -10,13 +10,13 @@ Kyma is an application runtime that provides you a flexible and easy way to conn
+Classic Kyma offered a fixed set of preconfigured components whose development rhythm is synchronized and determined by the release schedule. With the modular approach, Kyma components become modules, each providing one functionality developed independently of the other ones. With each successive release, fewer components will be available within the preconfigured Kyma, but more and more will be  ...
 
-The modular approach boosts Kyma's extensibility and scalability and makes it cost-effective. With fully modularized Kyma, you can choose from many of its modules to facilitate your business needs best.  You can enable or disable modules on demand and configure them, thus determining the costs generated by the infrastructure that your modules use.
+The modular approach boosts Kyma's extensibility and scalability, making it even more cost-effective. With fully modularized Kyma, you can choose from many of its modules to facilitate your business needs best.  You can enable modules on demand and configure them. By disabling the modules you don't need to use, you considerably reduce the attack surface area, as well as upgrade time and risk.
```
> how do I upgrade time? I associate either buying a fancy Swatch, or discovering in the area of Einstein/Hawking :D

I don't think you do, but you can reduce upgrade time (and risk) ;)

NHingerl replied:
:facepalm:
I reduce the *upgrade time*.

Maybe we could add a "the" to show that "upgrade" is not a verb here?

```suggestion
The modular approach boosts Kyma's extensibility and scalability, making it even more cost-effective. With fully modularized Kyma, you can choose from many of its modules to facilitate your business needs best.  You can enable modules on demand and configure them. By disabling the modules you don't need to use, you considerably reduce the attack surface area, as well as the upgrade time and risk.
```

IwonaLanger replied:
Good idea. Thanks for spotting the ambiguity 🙇‍♀️

IwonaLanger commented:
/retest

arturskorupa commented on docs/01-overview/README.md:
```diff
@@ -8,9 +8,22 @@ Kyma is an application runtime that provides you a flexible and easy way to conn
+
+### Kyma modules
+
+The first available Kyma modules are:
```
We say there are some modules.  Below particular functionalities are mentioned. How does the module differ from the functionality described below ? Maybe we should say it differently and just list all functionalities and say some of them are available in core Kyma, and some as module.

varbanv replied:
We have an explicit mention that the modules provide the functionality above: "Kyma components become modules, each providing one functionality developed independently"

varbanv replied:
For me, this section is to explicitly link to the relevant documentation for the modules. not to convey functionality. This has been a major requirement since the beginning - provide a table with the modules and links to their documentation.

arturskorupa commented on docs/01-overview/README.md:
```diff
@@ -8,9 +8,22 @@ Kyma is an application runtime that provides you a flexible and easy way to conn
 
+## Modular Kyma
+
+Classic Kyma offered a fixed set of preconfigured components whose development rhythm is synchronized and determined by the release schedule. With the modular approach, Kyma components become modules, each providing one functionality developed independently of the other ones. With each successive release, fewer components will be available within the preconfigured Kyma, but more and more will be  ...
```
I ask to myself is how to install Kyma. I find article under get started. But apparetnly it didn’t install both modules. I need a quidance how to do it. Probably it should be mentioned on quick install page (not only on a page  dedicated for  module).

mmitoraj replied:
I created this PR to complete the task - https://github.com/kyma-project/kyma/pull/17715. PTAL.

pbochynski reviewed (changes requested):
I see the content of the overview more like this:

# What is Kyma

Kyma is the opinionated set of Kubernetes based modular building blocks that includes the necessary capabilities to develop and run enterprise-grade cloud-native applications.
It is the open path to the SAP ecosystem supporting business scenarios end-to-end.

Kyma is the actively maintained open source project supported by SAP. Kyma projects is also a foundation of Kubernetes Runtime which is a part of SAP Business Technology Platform (BTP). You can use Kyma modules in your own Kubernetes cluster or you can try a managed version from SAP BTP with ready to use Kubernetes cluster (powered by Gardener). 

# Kyma modules

Classic Kyma runtime offered a fixed set of mandatory components whose development rhythm was synchronized and determined by the release schedule. With the modular approach, Kyma components become modules, each providing one functionality developed independently of the other ones. Each module comes with own custom resource that holds the desired configuration and the operator that reconciles the configuration. 
Kyma project is currently in the transition phase: some components are already independent modules, but other are still part of the big Kyma release and are installed with `kyma deploy` command. With each successive release, fewer components will be available within the preconfigured Kyma, but more and more will be offered as independent modules.

The list of Kyma modules (entries marked with “*” are still components and will be modularized soon):

| Module | Purpose | Documentation |
|---|---|---|
| BTP Operator |Within the BTP Operator module, BTP Manager installs SAP BTP Service Operator that allows you to consume SAP BTP services from your Kubernetes cluster using Kubernetes-native tools. | https://github.com/kyma-project/btp-manager |
| Istio* | Istio is a service mesh with Kyma-specific configuration. | https://github.com/kyma-project/istio |
| Serverless | With the Serverless module, you can define simple code snippets (Funtions) with minimal implementation effort. | https://github.com/kyma-project/serverless-manager |
| Telemetry* | Enable telemetry agents to easily collect application logs and distributed traces for your application and dispatch them to backends.|  https://github.com/kyma-project/telemetry-manager |
| Eventing* | Eventing provides functionality to publish and subscribe to CloudEvents. <br> At the moment, the SAP Event Mesh default plan and NATS (provided by the NATS module) are supported. | https://github.com/kyma-project/eventing-manager |
| NATS* |NATS deploys a NATS cluster within the Kyma cluster. You can use it as a backend for Kyma Eventing. | https://github.com/kyma-project/nats-manager |
| Application Connector* | Application Connector allows you to connect with external solutions. No matter if you want to integrate an on-premise or a cloud system, the integration process doesn't change, which allows you to avoid any configuration or network-related problems. | https://github.com/kyma-project/application-connector-manager
| API Gateway* |API Gateway provides functionalities that allow you to expose and secure APIs. |
| Keda | KEDA is a flexible Event Driven Autoscaler for the Kubernetes workloads. | https://github.com/kyma-project/keda-manager |


# FAQ

### Why should I use Kyma? What are the advantages?
Kyma is built upon leading cloud-native, open-source projects and open standards, such as Istio, NATS, Cloud Events, Open Telemetry, and Prometheus. We created an opinionated set of modules that you can easily enable in your Kubernetes cluster to speed up cloud application development and operations. With Kyma you save the time to pick the right tools, and the effort to keep them secure and up to date. 

### Is Kyma just another platform?

No, Kyma is not intended to be a full blown platform. It is the application runtime based on Kubernetes with several extensions. Those extensions make Kyma more attractive for developers that want to focus more on business logic and limit investment in technical services and infrastructure. Kyma is part of the SAP Business Technology Platform and offers easy integration with BTP services and other SAP systems.

### What is the difference between open source Kyma project and SPA BTP Kyma Runtime?

SAP BTP Kyma Runtime is a bundle of Kubernetes cluster (powered by Gardener) and Kyma modules provided as a managed service. All the components are regularly updated, and the availability is monitored and guaranteed (SLA). Managed Kyma runtime is also preconfigured to easily connect to other SAP services and systems. Using Kyma Runtime you can face some limitations in configuring Kyma components (some settings are managed centrally and overwrite user changes), but still customer gets the admin access to the cluster (cluster-admin role).
Using Kyma open source components you have more control and more flexibility over installation, configuration and upgrade process, but you have more responsibilities related to operations.

### Can I mix Kyma modules with other Kubernetes tools?

Yes. Kyma is opinionated but open. You can pick the modules you need from Kyma and complement them with other tools. 

### Kyma is a part of the commercial product (SAP BTP Kyma Runtime). What is the chance that Kyma will become closed project?

Kyma is open source since 2018 and it is part of SAP BTP since 2019. We believe that opennes and vendor independence is a value proposition. We also believe that offering open source project as a commercial product brings only benefits to both sides. Open source users get the confidence that the project won't be abandoned anytime soon, customers see the quality and technical details of the product. Apart from that SAP is a strong supporter of the open source community (see: https://community.sap.com/topics/open-source).

IwonaLanger commented:
> # FAQ
> ### Why should I use Kyma? What are the advantages?
> Kyma is built upon leading cloud-native, open-source projects and open standards, such as Istio, NATS, Cloud Events, Open Telemetry, and Prometheus. We created an opinionated set of modules that you can easily enable in your Kubernetes cluster to speed up cloud application development and operations. With Kyma you save the time to pick the right tools, and the effort to keep them secure and up to date.
> 
> ### Is Kyma just another platform?
> No, Kyma is not intended to be a full blown platform. It is the application runtime based on Kubernetes with several extensions. Those extensions make Kyma more attractive for developers that want to focus more on business logic and limit investment in technical services and infrastructure. Kyma is part of the SAP Business Technology Platform and offers easy integration with BTP services and other SAP systems.
> 
> ### What is the difference between open source Kyma project and SPA BTP Kyma Runtime?
> SAP BTP Kyma Runtime is a bundle of Kubernetes cluster (powered by Gardener) and Kyma modules provided as a managed service. All the components are regularly updated, and the availability is monitored and guaranteed (SLA). Managed Kyma runtime is also preconfigured to easily connect to other SAP services and systems. Using Kyma Runtime you can face some limitations in configuring Kyma components (some settings are managed centrally and overwrite user changes), but still customer gets the admin access to the cluster (cluster-admin role). Using Kyma open source components you have more control and more flexibility over installation, configuration and upgrade process, but you have more responsibilities related to operations.
> 
> ### Can I mix Kyma modules with other Kubernetes tools?
> Yes. Kyma is opinionated but open. You can pick the modules you need from Kyma and complement them with other tools.
> 
> ### Kyma is a part of the commercial product (SAP BTP Kyma Runtime). What is the chance that Kyma will become closed project?
> Kyma is open source since 2018 and it is part of SAP BTP since 2019. We believe that opennes and vendor independence is a value proposition. We also believe that offering open source project as a commercial product brings only benefits to both sides. Open source users get the confidence that the project won't be abandoned anytime soon, customers see the quality and technical details of the product. Apart from that SAP is a strong supporter of the open source community (see: https://community.sap.com/topics/open-source).

I'm not sure the overview is the right space for FAQ. We might have a separate doc for those.

pbochynski commented:
> I'm not sure the overview is the right space for FAQ. We might have a separate doc for those.

I replaced the advantages section with FAQ. It doesn't have to be named FAQ. I just wanted to put that message in the overview. It is rather not a full FAQ - it is more about why should I use Kyma and what it is. You can change it back from FAQ format to a regular chapter if you like.

mmitoraj commented on docs/README.md:
```diff
@@ -1,22 +1,34 @@
-Kyma is built upon leading cloud-native, open-source projects, such as Istio, NATS, Serverless, and Prometheus. The features developed by Kyma are the unique “glue” that holds them together, so you can connect and extend your applications easily and intuitively. To learn how to do that, head over to the [Get Started](./02-get-started) section where you can find step-by-step instructions to get yo ...
+| Module | Purpose |
+|---|---|
+| [BTP Operator]((https://kyma-docs.netlify.app//?basePath=https://raw.githubusercontent.com/kyma-project/btp-manager/aa50848013372806eaf2e707c217b8bed4eb09cb/docs/user/&homepage=README.md&sidebar=true&loadSidebar=_sidebar.md&browser-tab-title=BTP%20Operator%20Documentation#/)) |Within the BTP Operator module, BTP Manager installs SAP BTP Service Operator that allows you to consume SAP BTP servic ...
```
```suggestion
| [BTP Operator]((https://kyma-docs.netlify.app//?basePath=https://raw.githubusercontent.com/kyma-project/btp-manager/aa50848013372806eaf2e707c217b8bed4eb09cb/docs/user/&homepage=README.md&sidebar=true&loadSidebar=_sidebar.md&browser-tab-title=BTP%20Operator%20Documentation#/)) |Within the BTP Operator module, BTP Manager installs SAP BTP Service Operator that allows you to consume SAP BTP services from your Kubernetes cluster using Kubernetes-native tools. |
```
The link doesn't work.

mmitoraj commented on docs/README.md:
```diff
@@ -1,22 +1,34 @@
+| Module | Purpose |
+|---|---|
+| [BTP Operator]((https://kyma-docs.netlify.app//?basePath=https://raw.githubusercontent.com/kyma-project/btp-manager/aa50848013372806eaf2e707c217b8bed4eb09cb/docs/user/&homepage=README.md&sidebar=true&loadSidebar=_sidebar.md&browser-tab-title=BTP%20Operator%20Documentation#/)) |Within the BTP Operator module, BTP Manager installs SAP BTP Service Operator that allows you to consume SAP BTP servic ...
+| [Istio*](https://github.com/kyma-project/istio) | Istio is a service mesh with Kyma-specific configuration. | |
```
```suggestion
| [Istio*](https://github.com/kyma-project/istio) | Istio is a service mesh with Kyma-specific configuration. |
```

mmitoraj commented on docs/README.md:
```diff
@@ -1,22 +1,34 @@
+|---|---|
+| [BTP Operator]((https://kyma-docs.netlify.app//?basePath=https://raw.githubusercontent.com/kyma-project/btp-manager/aa50848013372806eaf2e707c217b8bed4eb09cb/docs/user/&homepage=README.md&sidebar=true&loadSidebar=_sidebar.md&browser-tab-title=BTP%20Operator%20Documentation#/)) |Within the BTP Operator module, BTP Manager installs SAP BTP Service Operator that allows you to consume SAP BTP servic ...
+| [Istio*](https://github.com/kyma-project/istio) | Istio is a service mesh with Kyma-specific configuration. | |
+| [Serverless](https://github.com/kyma-project/serverless-manager) | With the Serverless module, you can define simple code snippets (Funtions) with minimal implementation effort. |  |
```
```suggestion
| [Serverless](https://github.com/kyma-project/serverless-manager) | With the Serverless module, you can define simple code snippets (Functions) with minimal implementation effort. |
```

mmitoraj commented on docs/README.md:
```diff
@@ -1,22 +1,34 @@
+| [BTP Operator]((https://kyma-docs.netlify.app//?basePath=https://raw.githubusercontent.com/kyma-project/btp-manager/aa50848013372806eaf2e707c217b8bed4eb09cb/docs/user/&homepage=README.md&sidebar=true&loadSidebar=_sidebar.md&browser-tab-title=BTP%20Operator%20Documentation#/)) |Within the BTP Operator module, BTP Manager installs SAP BTP Service Operator that allows you to consume SAP BTP servic ...
+| [Istio*](https://github.com/kyma-project/istio) | Istio is a service mesh with Kyma-specific configuration. | |
+| [Serverless](https://github.com/kyma-project/serverless-manager) | With the Serverless module, you can define simple code snippets (Funtions) with minimal implementation effort. |  |
+| [Telemetry*](https://github.com/kyma-project/telemetry-manager) | Enable telemetry agents to easily collect application logs and distributed traces for your application and dispatch them to backends.|   |
```
```suggestion
| [Telemetry*](https://github.com/kyma-project/telemetry-manager) | Enable telemetry agents to easily collect application logs and distributed traces for your application and dispatch them to backends.|
```

mmitoraj commented on docs/README.md:
```diff
@@ -1,22 +1,34 @@
+| [Istio*](https://github.com/kyma-project/istio) | Istio is a service mesh with Kyma-specific configuration. | |
+| [Serverless](https://github.com/kyma-project/serverless-manager) | With the Serverless module, you can define simple code snippets (Funtions) with minimal implementation effort. |  |
+| [Telemetry*](https://github.com/kyma-project/telemetry-manager) | Enable telemetry agents to easily collect application logs and distributed traces for your application and dispatch them to backends.|   |
+| [Eventing*](https://github.com/kyma-project/eventing-manager) | Eventing provides functionality to publish and subscribe to CloudEvents. <br> At the moment, the SAP Event Mesh default plan and NATS (provided by the NATS module) are supported. |  |
```
```suggestion
| [Eventing*](https://github.com/kyma-project/eventing-manager) | Eventing provides functionality to publish and subscribe to CloudEvents. <br> At the moment, the SAP Event Mesh default plan and NATS (provided by the NATS module) are supported. |
```

mmitoraj commented on docs/README.md:
```diff
@@ -1,22 +1,34 @@
+| [Serverless](https://github.com/kyma-project/serverless-manager) | With the Serverless module, you can define simple code snippets (Funtions) with minimal implementation effort. |  |
+| [Telemetry*](https://github.com/kyma-project/telemetry-manager) | Enable telemetry agents to easily collect application logs and distributed traces for your application and dispatch them to backends.|   |
+| [Eventing*](https://github.com/kyma-project/eventing-manager) | Eventing provides functionality to publish and subscribe to CloudEvents. <br> At the moment, the SAP Event Mesh default plan and NATS (provided by the NATS module) are supported. |  |
+| [NATS*](https://github.com/kyma-project/nats-manager) |NATS deploys a NATS cluster within the Kyma cluster. You can use it as a backend for Kyma Eventing. |  |
```
```suggestion
| [NATS*](https://github.com/kyma-project/nats-manager) |NATS deploys a NATS cluster within the Kyma cluster. You can use it as a backend for Kyma Eventing. |
```

mmitoraj commented on docs/README.md:
```diff
@@ -1,22 +1,34 @@
 
-The extensions and customizations you create are decoupled from your core applications, which adds to these general advantages of using Kyma:
+### Kyma Beta modules:
+> **NOTE:** The entries marked with "*" are still components that will be modularized soon.
```
Do we need the NOTE? There are no components in the table.

IwonaLanger replied:
> Do we need the NOTE? There are no components in the table.

Just forgot to remove it. Thanks for spotting it :)

mmitoraj reviewed (commented):
Shouldn't the `docs/README.md` be the same as the `01-overview.README.md`? I'm not technically sure why we cannot pull from the same file but fro me they should present the same content.

IwonaLanger commented:
> My bad, I forgot about Keda. Added it.

I added it in another table to separate beta modules. Would you like to have them together in one table? If so, should we somehow indicate that it's beta?

IwonaLanger commented:
> Shouldn't the `docs/README.md` be the same as the `01-overview.README.md`? I'm not technically sure why we cannot pull from the same file but fro me they should present the same content.

I'm not sure about this one. @valentinvieriu says they don't have to be the same so I thought we might keep a bit more content in the `docs/01-overview/README.md`, but I won't insist on it.

mmitoraj commented on docs/README.md:
```diff
@@ -1,22 +1,34 @@
-Out of the box, Kyma offers various functionalities, such as:  
+## Modular Kyma
+
+Classic Kyma offered a fixed set of preconfigured components whose development rhythm is synchronized and determined by the release schedule. With the modular approach, Kyma components become modules, each providing one functionality developed independently of the other ones. Each module has its own custom resource that holds the desired configuration and the operator that reconciles the configur ...
```
I would also link to the `docs/frequently-asked-questions.md` from the overview page, what do you think?

pbochynski replied:
> I would also link to the `docs/frequently-asked-questions.md` from the overview page, what do you think?

My intention was to replace Advantages section but was blurry. I chose the FAQ format as more direct and easier to read  (IMHO). I wanted to keep it on that page as the page is relatively short after editing.

pbochynski commented:
> I added it in another table to separate beta modules. Would you like to have them together in one table? If so, should we somehow indicate that it's beta?

Beta is not applicable in the open source. We do not offer SLA, so no difference from other modules. We still use beta/alpha versions to describe API maturity (not only in Keda, other modules also use alpha/beta APIs).

NHingerl commented on docs/README.md:
```diff
@@ -1,22 +1,38 @@
-Kyma is an application runtime that provides you a flexible and easy way to connect, extend, and customize your applications in the cloud-native world of Kubernetes.
+## What is Kyma
```
The same suggestions apply as for the other README document.

NHingerl commented:
Do we really want to keep "Main Areas" as a navigation node? I thought we planned to get rid of it. The additional hierarchy level doesn't bring any value.
 
<img width="165" alt="image" src="https://github.com/kyma-project/kyma/assets/76950046/97a7a98d-8a1d-4e01-9dce-ec11b6a7833c">

IwonaLanger commented on docs/01-overview/README.md:
```diff
@@ -2,31 +2,39 @@
+It is the open path to the SAP ecosystem supporting business scenarios end-to-end.
 
-Kyma is an application runtime that provides you a flexible and easy way to connect, extend, and customize your applications in the cloud-native world of Kubernetes.
+Kyma is an actively maintained open-source project supported by SAP. The Kyma project is also a foundation of Kubernetes Runtime which is a part of SAP Business Technology Platform (BTP). You can use Kyma modules in your own Kubernetes cluster, or try the managed version from SAP BTP with a ready-to-use Kubernetes cluster powered by Gardener. 
```
> I don't really get this sentence:
> 
> > The Kyma project is also a foundation of Kubernetes Runtime which is a part of SAP Business Technology Platform (BTP).
> 
> Are we trying to say that the (OS) Kyma project is the foundation of the SAP BTP, Kyma runtime?
> 
> Because "Kubernetes Runtime", capitalized and without article, looks like it's an SAP product name, but I don't know any product with that name. I only know of SAP BTP, Kyma runtime.

Good point, thank you!

IwonaLanger replied:
Good point, thank you!

IwonaLanger commented:
> Do we really want to keep "Main Areas" as a navigation node? I thought we planned to get rid of it. The additional hierarchy level doesn't bring any value.
> 
> <img alt="image" width="165" src="https://user-images.githubusercontent.com/76950046/247911216-97a7a98d-8a1d-4e01-9dce-ec11b6a7833c.png">

Done :)

grego952 commented on docs/README.md:
```diff
@@ -1,22 +1,44 @@
+
+With the modular approach, Kyma components become modules, each providing one functionality developed independently of the other ones. Each module has its own custom resource that holds the desired configuration and the operator that reconciles the configuration. 
+
+The Kyma project is currently in the transition phase. Some components are already independent modules, but others are still part of the big Kyma release and are installed with `kyma deploy` command. With each successive release, fewer components will be available within preconfigured Kyma, but more and more will be offered as independent modules.
```
```suggestion
The Kyma project is currently in the transition phase. Some components are already independent modules, but others are still part of the big Kyma release and are installed with the `kyma deploy` command. With each successive release, fewer components will be available within preconfigured Kyma, but more and more will be offered as independent modules.
```

pbochynski commented:
I've noticed that we have the same (almost) content in docs/README.md and docs/01-overview/README.md 
Can we remove one of them?

IwonaLanger commented:
> I've noticed that we have the same (almost) content in docs/README.md and docs/01-overview/README.md Can we remove one of them?

I asked @valentinvieriu the same question, but he advised against removing either of the two files.

NHingerl commented on docs/01-overview/README.md:
```diff
@@ -5,9 +5,9 @@ title: What is Kyma?
+![overview](./assets/modular-kyma.png)
 
-![overview](./assets/kyma-overview.svg)
+Kyma is an actively maintained open-source project supported by SAP. The Kyma project is also a foundation of SAP BTP, Kyma Runtime which is a part of SAP Business Technology Platform (BTP). You can use Kyma modules in your own Kubernetes cluster, or try the managed version from SAP BTP with a ready-to-use Kubernetes cluster powered by Gardener. 
```
```suggestion
Kyma is an actively maintained open-source project supported by SAP. The Kyma project is also a foundation of SAP BTP, Kyma runtime which is a part of SAP Business Technology Platform (BTP). You can use Kyma modules in your own Kubernetes cluster, or try the managed version from SAP BTP with a ready-to-use Kubernetes cluster powered by Gardener. 
```
Official spelling is with lowercase "r": https://www.sapbrandtools.com/naming-center#/requests/10022

IwonaLanger commented on docs/01-overview/README.md:
```diff
@@ -5,9 +5,9 @@ title: What is Kyma?
 It is the open path to the SAP ecosystem supporting business scenarios end-to-end.
 
-Kyma is an actively maintained open-source project supported by SAP. The Kyma project is also a foundation of SAP BTP, Kyma Runtime which is a part of SAP Business Technology Platform (BTP). You can use Kyma modules in your own Kubernetes cluster, or try the managed version from SAP BTP with a ready-to-use Kubernetes cluster powered by Gardener. 
+![overview](./assets/modular-kyma.png)
```
> wow cool who created this image?! ❤️

Not me :) It's PB's creation. And it is cool, I agree :)

chrkl commented on installation/resources/crds/telemetry/logpipelines.crd.yaml:
```diff
@@ -341,7 +341,7 @@ spec:
               unsupportedMode:
                 description: Is active when the LogPipeline uses a `custom` output
-                  or filter; see [unsupported mode](./../../01-overview/main-areas/telemetry/telemetry-02-logs.md#unsupported-mode#unsupported-mode).
+                  or filter; see [unsupported mode](./../../01-overview/telemetry/telemetry-02-logs.md#unsupported-mode#unsupported-mode).
```
We should not use relative links in the CRD description since this is also used outside of the docs and will not work there. Can you fix this within this PR or should we create a follow-up PR?

IwonaLanger replied:
> We should not use relative links in the CRD description since this is also used outside of the docs and will not work there. Can you fix this within this PR or should we create a follow-up PR?

TBH, I'd prefer to do it in another PR.

grego952 commented:
/retest

mvshao commented:
/test pre-main-kyma-integration-k3d-runtime-agent
